testing/regress/ecl/dropzone_query_test.ecl

```diff
@@ -0,0 +1,7 @@
+//nokey
+
+import Std.File AS FileServices;
+
+output(FileServices.GetDefaultDropZone());
+
+output(FileServices.GetDropZones());
```

Comment:
There doesn't appear to be an accompanying key file (dropzone_query_test.xml) ?
But if that's the case, why isn't the PR failing?